### DISCUSSION TITLE
Change camera time

### DISCUSSION THREAD

**Discussion by Imwithstupid007** [blakeblackshear/frigate#13783] [Ask A Question] [answered]
Just a short question to enhance my understanding of Frigate.
In a running Frigate system, are there consequences when changing the camera time?

A little context:
I noticed that one of my cameras time was off (it ran a few minutes late) and it didn't use my NTP server. So I corrected this (say at 18:35). Probably because of this I couldn't view recordings of the specific camera during that full hour (18:00-19:00). After the hour passed (after 19:00) recordings for that camera where normal again.

On a side note; are there recommendations on the use of a NTP server for cameras and NAS/Frigate?

TIA

**Comment by NickM-27** [comment, marked as answer]
yes, if the camera time is off it can affect certain things with the cameras timestamps causing stream instability.

I have all cameras blocked off from WAN but they can talk to the router itself on the ntp port
